### ISSUE TITLE
EBR_100_Gomez dialogue DIA_Gomez_Hello should not be available after Chapter 2.

### ISSUE THREAD

**Issue by Fawkes-dev** [Fawkes-dev/G1-CZ-Revision#57]
As soon as player joins any camp in Chapter 2 this dialogue should not be available anymore, otherwise you can offer services to Gomez in later chapters as well:
https://github.com/Fawkes-dev/G1-CZ-Revision/blob/96d79a08cc1e6dfd36180ee7e767cbc0d598d592/content/Story/Missions/DIA_EBR_100_Gomez.d#L70-L88

**Comment by AmProsius** [comment]
I think it would be better to check if player has `GIL_NONE` and not make this chapter specific.

**Comment by Fawkes-dev** [comment]
You are right, at first I was thinking `GIL_NONE` cannot be an option - because in Ch. 4 player (if member of the Old Camp) goes back to `GIL_NONE`. However at the same time, the only way player joins the Old camp is via `GIL_STT`, where he has to go through this dialogue. This dialogue increases value of variable `gomez_kontakte` - will be `> 3` and therefore dialogue won't be available.
